packages/apps/job-launcher/server/src/modules/auth/auth.service.ts

```diff
@@ -190,10 +196,10 @@ export class AuthService {
   }
 
   public async restorePassword(data: RestorePasswordDto): Promise<void> {
-    const tokenEntity = await this.tokenRepository.findOne({
-      uuid: data.token,
-      tokenType: TokenType.PASSWORD,
-    });
+    const tokenEntity = await this.tokenRepository.findOneByUuidAndTokenType(
```

Comment:
Should it use tokenService instead of tokenRepository?

Reply:
I think we need to follow the logic: if a method that calls another service method can directly call a repository method, if that service method doesn't contain any business logic, just, for example, look up something in the DB and return it. @Dzeranov @portuu3

Reply:
You're right @eugenvoronov, that's the logic we're following to reduce the amount of code we have